docs/components/Header.js

```diff
@@ -34,11 +36,16 @@ function Header() {
               <Flex alignItems="center" gap={2}>
                 <GestaltLogo height={40} width={40} />
                 {/* slight tweak to vertically center to logo */}
-                <Box dangerouslySetInlineStyle={{ __style: { marginBottom: '1px' } }}>Gestalt</Box>
+                <Box dangerouslySetInlineStyle={{ __style: { marginBottom: '1px' } }}>
+                  Gestalt Design System
+                </Box>
               </Flex>
             </Box>
           </Link>
         </Text>
+        <Text size="sm" weight="bold">
+          v{GestaltPackageJson.version}
```

Comment:
@ayeshakmaz should we add spacing between the version number and the search field?
![image](https://user-images.githubusercontent.com/127199/156201556-3352d22d-68f9-4940-a2ec-f3fa5d6f5abb.png)